gazelle/internal/swiftcfg/swift_config.go

```diff
@@ -43,8 +43,13 @@ type SwiftConfig struct {
 
 	GenerateSwiftDepsForWorkspace bool
 
+	// The naming convention to apply to the module names derived from the directory names.
+	// The default behavior uses the name verbatim while PascalCase will convert snake_case to PascalCase.
 	ModuleNamingConvention string
 
+	// Whether the default behavior to tag generated swift library targets as manual should be disabled.
+	SwiftLibraryManualTagDisabled bool
```

Comment:
I wonder if we could instead allow configuring the tags that get added. Using none, you'd get this behavior but it's extensible to custom tags too. 

Not super familiar with what's allow in these tags though

Reply:
A client should not need to add the tag. In fact, I would seriously discourage it for most situations. Do you think this will be confusing for clients who see it in the doc?

Reply:
@cgrindel @luispadron I kinda like the idea of giving users an option to specify a list of tags to apply to generated targets, with "manual" being the default. I could imagine someone might want to add a "generated" tag or something like that.